run/test/specs/network_page_link_staking.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { englishStripped } from '../../localizer/Localizer';
+import { bothPlatformsIt } from '../../types/sessionIt';
+import { USERNAME } from '../../types/testing';
+import { SafariAddressBar, URLInputField } from './locators/browsers';
+import {
+  OpenLinkButton,
+  SessionNetworkLearnMoreStaking,
+  SessionNetworkMenuItem,
+} from './locators/network_page';
+import { UserSettings } from './locators/settings';
+import { isChromeFirstTimeOpen } from './utils/chrome_first_time_open';
+import { newUser } from './utils/create_account';
+import { closeApp, openAppOnPlatformSingleDevice, SupportedPlatformsType } from './utils/open_app';
+
+bothPlatformsIt({
+  title: 'Network page learn more staking link',
+  risk: 'medium',
+  testCb: networkPageLearnMore,
+  countOfDevicesNeeded: 1,
+});
+
+async function networkPageLearnMore(platform: SupportedPlatformsType) {
+  const { device } = await openAppOnPlatformSingleDevice(platform);
+  const linkURL = 'https://docs.getsession.org/session-network/staking';
+  await newUser(device, USERNAME.ALICE);
+  await device.clickOnElementAll(new UserSettings(device));
+  await device.onAndroid().scrollDown();
+  await device.clickOnElementAll(new SessionNetworkMenuItem(device));
+  await device.clickOnElementAll(new SessionNetworkLearnMoreStaking(device));
+  await device.checkModalStrings(
+    englishStripped('urlOpen').toString(),
+    englishStripped('urlOpenDescription').withArgs({ url: linkURL }).toString()
+  );
+  await device.clickOnElementAll(new OpenLinkButton(device));
+  if (platform === 'ios') {
+    // Tap the Safari address bar to reveal the URL
+    await device.clickOnElementAll(new SafariAddressBar(device));
+  } else {
+    // Chrome can throw some modals on first open
+    await isChromeFirstTimeOpen(device);
+  }
+  const urlField = await device.waitForTextElementToBePresent(new URLInputField(device));
+  const actualUrlField = await device.getTextFromElement(urlField);
+  // Add https:// to the retrieved URL if the UI doesn't show it (Chrome doesn't, Safari does)
+  const fullRetrievedURL = actualUrlField.startsWith('https://')
+    ? actualUrlField
+    : `https://${actualUrlField}`;
```

Comment:
can you make a utility function to do this?  and replace its use in this file and  `run/test/specs/network_page_link_network.spec.ts`